packages/serverless-components/nextjs-component/src/component.ts

```diff
@@ -2,13 +2,13 @@ import { Component } from "@serverless/core";
 import { readJSON, pathExists } from "fs-extra";
 import { resolve, join } from "path";
 import { Builder } from "@sls-next/lambda-at-edge";
-import {
+import type {
```

Comment:
This was pulling from `lambda-at-edge/types.d.ts` now it will be from `lambda-at-edge/dist/index.ts`